ansible/.env.j2

```diff
@@ -0,0 +1,17 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=192b9bdd22ab9ed4d12e236c78afcb9a393ec15f71bbf5dc987d54727823bcbf
+
+# Trello
+TRELLO_BOARD_ID=65b7c6467d53453f75bfabf6
+TRELLO_TODO_LIST_ID=65b7c6467d53453f75bfabfd
+TRELLO_DONE_LIST_ID=65b7c6467d53453f75bfabff
+TRELLO_DOING_LIST_ID=65b7c6467d53453f75bfabfe
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)